error_test.rb

```diff
@@ -0,0 +1,71 @@
+require_relative "environment.rb"
+
+class ErrorTest
+  include Environment
+  attr_accessor :board, :depot, :control, :dictionary, :poker
+
+  def initialize
+    @board = {
+      players: {},
+      control: {},
+      status: nil
+    }
```

Comment:
When possible try to save lines on the code without killing how readable it is, I would set this kind of variables as:
`@board = { players: {}, control: {}, status: nil }`